apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -74,7 +74,7 @@ export const detectTargetedSections = (newsItemScopes: NewsItemScope[], property
 const areTargetedToOneProperty = (newsItemScopes): boolean => uniq(map(newsItemScopes, ['property', 'id'])).length === 1
 
 const buildMessageFromNewsItemScopes = (newsItemScopes, intl): string => {
-    if (newsItemScopes.length === 0) {
+    if (newsItemScopes.length === 1 && get(newsItemScopes[0], 'property') === null) {
```

Comment:
You can shorten this expression with lodash `get`
```suggestion
    if (get(newsItemScopes, '0.property') === null) {
```